### ISSUE TITLE
Optional `Char` fields with choices convert to required enum.

### ISSUE THREAD

**Issue by Eraldo** [graphql-python/graphene-django#637]
I ran into this closed issue and it still seems to be an issue:
https://github.com/graphql-python/graphene-django/pull/525#issuecomment-431182216

Related:
https://github.com/graphql-python/graphene-django/issues/503
https://github.com/graphql-python/graphene-django/issues/67

I found this like to be the hinge:
https://github.com/graphql-python/graphene-django/blob/dc561c68c49d1a91637503f765857d819c36769a/graphene_django/converter.py#L74

Proposed solution...
How about changing it to:
```
converted = enum(description=field.help_text, required=not (field.null or field.blank))
```

**Comment by Eraldo** [comment]
Any comment to my suggestion?

**Comment by zbyte64** [comment]
Duplicate of #503 we have a fix pr at #714 ; you can install that branch for a fix
